core-services/release-controller/_releases/release-ocp-4.6-ci.json

```diff
@@ -116,7 +116,7 @@
       "prowJob":{"name":"release-openshift-origin-installer-e2e-azure-ovn-upgrade-4.5-stable-to-4.6-ci"}
     },
     "upgrade-ovirt-minor":{
-      "cron":"@every 12h",
+      "cron":"0 6 19 * *",
```

Comment:
This value is different than the rest.  Is that intentional?  Should it be "0 6,19 * * *"?